services/language_translator/v3-doc.js

```diff
@@ -17,10 +17,11 @@
 module.exports = function (RED) {
   const request = require('request'),
     SERVICE_IDENTIFIER = 'language-translator',
-    SERVICE_VERSION = '2018-05-01';
+    SERVICE_VERSION = '2018-05-01',
+    LanguageTranslatorV3 = require('ibm-watson/language-translator/v3'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)